### PR TITLE
(core) show next scheduled run for cron triggered pipelines

### PR DESCRIPTION
**Pull request by anotherchrisberry**
screen shot 2016-08-12 at 12 19 57 pm

### CODE REVIEW

**Comment by ajordens** [comment]
Should we just do this server-side and return as an attribute of the cron trigger?

```
CronTrigger tr = new CronTrigger();
tr.setCronExpression("0 0 23 3,18 * ? *");
Date nextFireAt = tr.getFireTimeAfter(currTime);
```

**Comment by anotherchrisberry** [comment]
If we want to do it server-side, we have to take into consideration a couple of things:
1. the trigger is defined in a specific timezone, not UTC
2. we use the quartz scheduler, not the standard CRON trigger, so days of the week are 1-based, not zero-based.

**Comment by tomaslin** [comment]
should it be next to enabled so it says `Trigger : enabled 🕐 `? This would make it clearer that it's related to the trigger.

File: app/scripts/modules/core/delivery/triggers/nextRun.component.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      pipeline: '<'
+    },
+    controller: function (momentService, settings) {
+
+      function setScheduled() {
+        if (!this.pipeline) {
+          return;
+        }
+        let [cron] = this.pipeline.triggers.filter(t => t.type === 'cron' && t.enabled);
+        if (cron) {
```
Comment (tomaslin) [inline app/scripts/modules/core/delivery/triggers/nextRun.component.js]: does this handle having 2 cron triggers?

**Comment by anotherchrisberry** [comment]
screen shot 2016-08-12 at 1 04 13 pm
I could go either way on this one...

File: app/scripts/modules/core/delivery/triggers/nextRun.component.js (hunk 1 above)
Comment (ajordens) [inline app/scripts/modules/core/delivery/triggers/nextRun.component.js]: should only report whichever cron trigger is next to fire?

**Comment by tomaslin** [comment]
at least it doesn't look like the icons we have next to configure / start execution this way

File: app/scripts/modules/core/delivery/triggers/nextRun.component.js (hunk 1 above)
Comment (anotherchrisberry) [inline app/scripts/modules/core/delivery/triggers/nextRun.component.js]: Ugh, it does now, although shame on whoever configured multiple cron triggers for their pipeline.

**Pull request merged**